### PR TITLE
Fix MicroOS51 profile for new kiwi

### PR DESCRIPTION
**Pull request by nadvornik**
New kiwi does not accept `devicepersistency="by-path"`. This changes it to `devicepersistency="by-uuid"` in MicroOS51 profile.
It also updates the list of available profiles.

### CODE REVIEW

File: OSImage/README.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ Available Kiwi profiles:
   * `Default-kvm`
   * `Default-dasd`
   * `Default-fba`
```
Comment (aaannz) [inline OSImage/README.md:45]: Can you remove these profiles? All of them are for s390 and I am not sure we support s390 build hosts and thus image building.

**Review by aaannz — APPROVED** [review]
LGTM, just one request wrt s390.

File: OSImage/README.md (hunk 1 above)
Comment (nadvornik) [reply OSImage/README.md:45]: I want to keep the profile the same as in Build Service.

Comment (nadvornik) [reply OSImage/README.md]: ok, done

**Pull request merged**